src/__test__/components/data-processing/data-integration/DataIntegration.test.jsx

```diff
@@ -21,6 +21,7 @@ const dataIntegrationFrequencyConfig = initialPlotConfigStates.dataIntegrationFr
 const dataIntegrationElbowConfig = initialPlotConfigStates.dataIntegrationElbow;
```

Comment:
do we have a unit test to check if data integration step indeed uses the data produced from configure embedding step?

Reply:
the new changes to the tests in data integration ensure that the embedding uses them since now the plotUuid for the embedding plot is the one associated to Configure Embedding.

What I will add is a test to make sure that plots are blocked when embedding is still not completed.

Reply:
ok, that sounds like a good idea! 👍